app/controllers/claims/schools/claims_controller.rb

```diff
@@ -85,4 +87,8 @@ def claim_id
   def authorize_claim
     authorize @claim || Claims::Claim
   end
+
+  def has_school_accepted_grant_conditions?
+    redirect_to claims_school_accept_grant_conditions_path(school_id: @school) if @school.grant_conditions_accepted_at.nil?
+  end
```

Comment:
This seems to be repeated a lot. Maybe the logic here should be DRYed up into either a concern or a shared inherited controller, `Claims::ApplicationController`.

I think the logic can also be cleaned up into:

```suggestion
  def has_school_accepted_grant_conditions?
    redirect_to claims_school_accept_grant_conditions_path(@school) if @school.grant_conditions_accepted?
  end
```

And we create a boolean reader:

```ruby
class Claims::School
  def grant_conditions_accepted?
    grant_conditions_accepted_at?
  end

  # This could also be
  alias_method :grant_conditions_accepted_at?, :grant_conditions_accepted?
end
```